ucm2/Qualcomm/x1e80100/HiFi.conf

```diff
@@ -67,3 +81,48 @@ SectionDevice."Mic" {
 		CapturePCM "hw:${CardId},3"
 	}
 }
+
+SectionDevice."DisplayPort0" {
```

Comment:
Maybe this should be called HDMI? https://www.alsa-project.org/alsa-doc/alsa-lib/group___use.html#gab4e4a2b3d44883b1ffb5406529746faf Although technically it is USB/DisplayPort.